docs/DeveloperGuide.md

```diff
@@ -494,11 +494,13 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Non-Functional Requirements
 
-1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-
-*{More to be added}*
+1. Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
+2. Should be able to hold up to 1000 patients without noticeable sluggishness in performance for typical usage.
+3. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+4. Should not require an external installer or launcher.
+5. Data should be stored locally in an easily-editable text file.
+6. Should not depend on any cloud-based remote server.
+7. Data should not be stored in an external Database Management System (DBMS) or data warehouse. 
```

Comment:
Double check if the numbering requires double spacing between the full stop and the beginning of the sentence? Seems like a common pattern in lists elsewhere in the document.

Reply:
Fixed in latest commit. Thanks.